README.md

```diff
@@ -37,8 +37,29 @@ User: A person that uses our software.
 
 Cantis also generates its own [glossary](glossary.txt).
 
+## Install
+In order to use the `@GlossaryTerm` annotation, you must add a dependency to `cantis` to your project:
+
+Maven:
+
+```xml
+<dependency>
+    <groupId>com.github.korthout</groupId>
+    <artifactId>cantis</artifactId>
+    <version>0.1</version>
+</dependency>
+```
+
+Gradle:
+
+```
+dependencies {
+    implementation 'org.hibernate:hibernate-core:3.6.7.Final'
```

Comment:
I like the idea to support Gradle too. I guess this would work, but I've yet to try it out.
```suggestion
    implementation 'com.github.korthout:cantis:0.1'
```